xchange-bitmex/src/main/java/info/bitrich/xchangestream/bitmex/BitmexStreamingMarketDataService.java

```diff
@@ -89,4 +106,42 @@ public BitmexStreamingMarketDataService(BitmexStreamingService streamingService)
             return trades;
         });
     }
+
+
+    public Observable<BitmexExecution> getExecutions(String symbol) {
```

Comment:
Rename to `getRawExecutions` for consistency with other exchanges, and receive `CurrencyPair` instead of symbol (they are equivalent now - see above).